feature/mypage/src/main/java/com/linku/mypage/MyPageViewModel.kt

```
@@ -147,21 +167,12 @@ class MyPageViewModel @Inject constructor(
         viewModelScope.launch {
             try {
                 userRepository.logout()
+                _uiState.value = MyPageUiState()
                 onSuccess()
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)